pkg/install/1-installresources.go

```diff
@@ -742,7 +742,7 @@ func (i *Installer) installResources(ctx context.Context, doc *api.OpenShiftClus
 			}
 
 			if time.Now().Sub(now) > 30*time.Minute {
-				return fmt.Errorf("timed out waiting for bootstrap configmap")
+				return fmt.Errorf("timed out waiting for bootstrap configmap. Last error: %v", err)
```

Comment:
@jim-minter PTAL. It was useful to know that master didn't come up (timeout after 30 minutes) when I was tryign to figure out what was going on with cluster creation.

I was initially thinking about doing the following somewhere before `<-t.C`:

```go
if err != nil {
	i.log.Infof("found error while waiting for bootstrap configmap: %#v. Retrying in %.0fs", err.Error(), tickerDuration.Seconds())
}
```

But, I think, it will make logs very noisy: it will log errors every 10 seconds for about 15 minutes even in for successful creation.